pkg/scheduler/api/job_info.go

```diff
@@ -190,6 +190,7 @@ func (ti *TaskInfo) Clone() *TaskInfo {
 		Name:           ti.Name,
 		Namespace:      ti.Namespace,
 		Priority:       ti.Priority,
+		PodVolumes:     ti.PodVolumes,
```

Comment:
should we use deep-copy?

Reply:
We did not mutate PodVolumes directly, so l think it ok to not use deepcopy here